src/middlewares/authMiddlewares.js

```
@@ -0,0 +1,25 @@
+const jwt = require('jsonwebtoken');
+const JWT_SECRET = process.env.JWT_SECRET;
```

`JWT_SECRET` is read from the environment without a fallback. In other files (`auth.controller.js`) a fallback/default value (`'supersecretkey'`) is used. This inconsistency can lead to `jwt.verify` failing unexpectedly in this middleware if `process.env.JWT_SECRET` is undefined. Options:
- Provide the same default here (e.g. `const JWT_SECRET = process.env.JWT_SECRET || 'supersecretkey'`) to match `auth.controller.js`, or
- Throw a clear error at startup if `JWT_SECRET` is not set so the app fails fast and you don't get silent verification issues.
Keeping the secret handling consistent prevents hard-to-diagnose token verification problems.